### ISSUE TITLE
Feature request: Add dropdown box to allow encoding selection

### ISSUE THREAD

**Issue by moonlightz** [clsid2/mpc-hc#2878]
Yeah.. it's me again.
The last time we spoke, I "complained" about the tv channels list not properly handling encoding.
It's been working since the change was made. The á é í... appeared alright until... ~3 weeks ago.

I usually use to grab EPG daily with a tool and I started to notice that there was some weird characters where supposed to be letters with accents. TV also got affected and everything that used tv too.
MPC-Hc was also affected.
I opned MPC-HC, gone to options > capture, ok button.
Then clicked File > Open device... , I can see the list of channels and it's all right. 
Then I clicked Scan button and clicked the Start button.
And I can could confirm.... the EPG/channels were no longer being supplied as UTF-8.
The í was abing switched by a T with a dash, and so on.
I complained to my isp and apparently they are clueless too.

So I wonder....
Is it possible to add a dropdown box with a encodings list like iso8859-1.....  to the Tuning dialog?
We could use this dropdown box to properly switch to the correct encoding.

**Comment by adipose** [comment]
>Is it possible to add a dropdown box with a encodings list like iso8859-1..... to the Tuning dialog?

Honestly, unlikely I'd get to that.  It's such a small group of people using DVB in the first place.  But are you saying it's Latin1?  If it's always latin-1 or UTF-8, maybe something can be done.

**Comment by clsid2** [comment]
It can be any of the 8859 variants.

Since it worked before, I would say issue is with provider, and not worth handling on player side.

**Comment by adipose** [comment]
Yeah I recall it actually identifies the encoding itself.  So if it is wrong then somebody screwed up.

**Comment by moonlightz** [comment]
Not sure what they did to the EPG service but it's kinda broken.
I been in touch to the some people and they pointed it's using iso8859-1.
A wrong move. UTF-8 is pratically universal unless they did gone to iso for compatibility.
The EPG can be extracted with EPG tools. I extracted EPG with DVBGuide and it can generate a xml file.
The way things appeared screwed in this xml file, it also appears with MPC-HC channels list.
Few days ago, I installed notepad++ on this system, loaded the xml file and switched it to iso..... and... it doesn't appear right.
No matter which one I picked, never gets it right.

I understand all what you wrote.
Maybe... you want to undo the utf-8 code you applied last time and then when a next release gets out, I will check if it gets right.
